hi-ml-histopathology/environment.yml

```diff
@@ -0,0 +1,73 @@
+name: HIMLHistopathology
+channels:
+  - defaults
+  - pytorch
+  - conda-forge
+dependencies:
+  - cudatoolkit=11.1
+  - pip=20.1.1
+  - python=3.7.3
+  - pytorch=1.10.0
+  - python-blosc==1.7.0
+  - torchvision=0.11.1
+  - pip:
+      - git+https://github.com/analysiscenter/radio.git@6d53e25#egg=radio
```

Comment:
I think a few packages can be cleaned up here: radio, Innereye-dicomrt, h5py, gputil?, numba, simpleitk, seaborn. Also, isnt' there hi-ml as a dependency?